docs/howto/manage-web-app-charms/configure-web-app-charms.rst

```diff
@@ -1,73 +1,10 @@
-.. _manage-12-factor-app-charms:
-
-
-Manage 12-factor app charms
-===========================
-
-    See also: :external+juju:ref:`Juju | Charm taxonomy <charm-taxonomy>`
-
-
-Prepare an OCI image for a 12-factor app charm
-----------------------------------------------
-
-    See more:
-    :external+rockcraft:doc:`Rockcraft | How to build a 12-factor app rock
-    <how-to/build-a-12-factor-app-rock>`
-
-
-Initialise a 12-factor app charm
---------------------------------
-
-Use ``charmcraft init`` and specify the relevant profile:
-
-.. code-block:: bash
-
-  charmcraft init --profile <profile>
-
-Charmcraft automatically creates a ``charmcraft.yaml`` project file, a
-``requirements.txt`` file and source code for the charm in your current directory. You
-will need to check the project file and ``README.md`` to verify that the charm's name
-and description are correct.
-
-    See also: :ref:`ref_commands_init`
-
-.. tabs::
-
-    .. group-tab:: Flask
-
-        Specify the ``flask-framework`` profile:
-
-        .. code-block:: bash
-
-            charmcraft init --profile flask-framework
-
-    .. group-tab:: Django
-
-        Specify the ``django-framework`` profile:
-
-        .. code-block:: bash
-
-            charmcraft init --profile django-framework
-
-    .. group-tab:: FastAPI
-
-        Specify the ``fastapi-framework`` profile:
-
-        .. code-block:: bash
-
-            charmcraft init --profile fastapi-framework
-
-    .. group-tab:: Go
-
-        Specify the ``go-framework`` profile:
-
-        .. code-block:: bash
-
-            charmcraft init --profile go-framework
-
+.. _configure-12-factor-charms:
```

Comment:
As mentioned elsewhere, the ref and filename are significant departures from the human title.